src/utils/client.js

```diff
@@ -33,7 +35,7 @@ async function onFirefoxConnect(actions) {
   }
 }
 
-async function onConnect(connection, actions) {
+async function onConnect(connection: Object, actions: Object) {
```

Comment:
Also, we don't need to type `actions` because it's the object that holds all of the actions, which is currently untyped